components/automate-ui/src/app/modules/infra-proxy/data-bags-details/data-bags-details.component.scss

```diff
@@ -256,6 +254,7 @@ tbody > .detail-row {
 }
 
 .float-right {
+  cursor: pointer;
   position: relative;
   float: right;
 }
```

Comment:
I'd prefer that we don't add a non-utility style to a css class that has a utility name.  What I mean by this is that the class name `float-right` makes sense when it is solely used to float an element to the right.  In this case, we are now doing more than just floating it.  Lets either change the class name OR apply the cursor pointer via its own CSS rule.